lib/iron_bank/user.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module IronBank
+  # A Zuora user, only available if the user data has been exported and provided
+  # to IronBank through the `zuora_users_file` configuration option.
+  #
+  # Cf. https://knowledgecenter.zuora.com/CF_Users_and_Administrators/A_Administrator_Settings/Manage_Users#Export_User_Data_to_a_CSV_File
+  #
+  class User
+    extend IronBank::Local
+
+    class << self
+      def store
+        zuora_users_file = IronBank.configuration.zuora_users_file
```

Comment:
WDYTA about setting up a delegator via [Forwardable](https://ruby-doc.org/stdlib-2.6.4/libdoc/forwardable/rdoc/Forwardable.html)?

Reply:
Add to use [`SingleForwardable`](https://ruby-doc.org/stdlib-2.5.1/libdoc/forwardable/rdoc/SingleForwardable.html) but yes, great suggestion! 👍